### PR TITLE
[#1993] Datepicker - disable year/month navigation if min date specified 

### PR DESCRIPTION
**Pull request by ebreaur**
I am trying to determine in _refreshView for all 3 (day/month/year) pickers if some navigation should be disabled.
After I set the result in the scope I am using the result in the templates like this: ng-disabled="navPrevDisabled" .

I am just disabling the buttons for now but if the approach is correct I can prevent it from keyboard as well hooking into 'move()' .

### CODE REVIEW

**Comment by barker1889** [comment]
:+1:

File: src/datepicker/datepicker.js [hunk 1]
```diff
@@ -246,6 +246,19 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
...
+        var calendarMinDate = new Date(firstDayOfMonth.getTime());
+        calendarMinDate.setHours(0);
+        // if the first day of the month is not greater then minDate there is no point to navigate to a previous month
+        scope.navPrevDisabled = ctrl.minDate && (calendarMinDate <= ctrl.minDate);
+        // compare maxDate with the first day of the next month
+        var calendarMaxDate = new Date(firstDayOfMonth.getTime());
+        calendarMaxDate.setHours(0);
+        calendarMaxDate.setMonth(calendarMaxDate.getMonth() + 1);
+        // if the first day of the next month is greater then maxDate there is no point to navigate to the next month
+        scope.navNextDisabled = calendarMaxDate > ctrl.maxDate;
```
Comment (jnorkus) [inline src/datepicker/datepicker.js]: Nav next will always be disabled if ctrl.maxDate == null

**Comment by karianna** [comment]
@ebreaur Are you able to rebase this PR?

**Comment by wesleycho** [comment]
Closing due to age of PR - feel free to open up a new PR with all conflicts fixed.

**Pull request closed without merging**